src/redux/actions/samples/createSample.js

```diff
@@ -38,7 +40,7 @@ const createSample = (
   let body;
 
   const newSample = {
-    ...sampleTemplate,
+    ..._.cloneDeep(sampleTemplate),
```

Comment:
no, because later on we are doing this:

```
filesToUploadV2Only.forEach((fileName) => {
      newSample.files[fileName] = { upload: { status: UploadStatus.UPLOADING } };
    });
```

and it actually writes those files into the `sampleTemplate` until we refresh

Reply:
Ok. I thought spreading copies the object deeply. It seems that nested objects are passed by reference and not copied.